File: example1/jdoodle.cs
using System;
using System.IO;
using iText.Kernel.Pdf;
using iText.Layout;
using iText.Layout.Element;

class Program
{
  static void Main() {
    PdfDocument pdfDocument = new PdfDocument(new PdfWriter(new FileStream("/myfiles/my.pdf", FileMode.Create, FileAccess.Write)));
    Document document = new Document(pdfDocument);
    
    String line = "Hello! Welcome to iTextPdf Git Test";
    document.Add(new Paragraph(line));
    document.Close();
  }
}

File: example1/jdoodle.java
import com.itextpdf.kernel.pdf.*;
import com.itextpdf.layout.Document;
import com.itextpdf.layout.element.Paragraph;
import java.io.*;

public class HelloWorld {
    public static final String DEST = "/myfiles/hello.pdf";

    public static void main(String args[]) throws IOException {
        File file = new File(DEST);
        file.getParentFile().mkdirs();
        new HelloWorld().createPdf(DEST);
    }

    public void createPdf(String dest) throws IOException {
        PdfDocument pdf = new PdfDocument(new PdfWriter(dest));
        Document document = new Document(pdf);

        String line = "Hello! Welcome to iTextPdf Git Test";
        document.add(new Paragraph(line));

        document.close();
    }
}
